### PR TITLE
remove 'automatically generated' strings from repo

### PR DESCRIPTION
**Pull request by joelanford**
Maintainers use an import organizer called 'gci' that ignores files with the string 'automatically generated' when called with its --skip-generated flag.

This commit is the result of the following command:

```
rg ' automatically generated' -l | xargs -n1 sed -i 's/ automatically generated//g'
```

None of the changed files are _actually_ generated files. We do not want gci to think they are.

**Description of the change:**

**Motivation for the change:**

**Reviewer Checklist**
- [ ] Implementation matches the proposed design, or proposal is updated to match implementation
- [ ] Sufficient unit test coverage 
- [ ] Sufficient end-to-end test coverage
- [ ] Docs updated or added to `/docs` 
- [ ] Commit messages sensible and descriptive

### CODE REVIEW

File: pkg/configmap/testdata/bundles/etcd.0.9.2/manifests/etcdoperator.v0.9.2.clusterserviceversion.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
 
     ### Reading and writing to etcd
 
-    Communicate with etcd though its command line utility `etcdctl` or with the API using the automatically generated Kubernetes Service.
```
Comment (grokspawn) [inline pkg/configmap/testdata/bundles/etcd.0.9.2/manifests/etcdoperator.v0.9.2.clusterserviceversion.yaml:20]: Is there any cause for concern that these diverge from the [source](https://github.com/operator-framework/community-operators/blob/1591b5e18d47129ed14e83ac2f9832d6cb41bb40/community-operators/etcd/0.9.4/etcdoperator.v0.9.4.clusterserviceversion.yaml#L166)

Comment (joelanford) [reply pkg/configmap/testdata/bundles/etcd.0.9.2/manifests/etcdoperator.v0.9.2.clusterserviceversion.yaml:20]: I don't think so, all just one-time copied test files.

Comment (grokspawn) [reply pkg/configmap/testdata/bundles/etcd.0.9.2/manifests/etcdoperator.v0.9.2.clusterserviceversion.yaml:20]: Also :whistles: .... the etcd content is duplicated all over this repo.

Comment (joelanford) [reply pkg/configmap/testdata/bundles/etcd.0.9.2/manifests/etcdoperator.v0.9.2.clusterserviceversion.yaml:20]: Yeah, it's... messy.

**Review by grokspawn — APPROVED** [review]

**Pull request merged**